xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -540,6 +540,9 @@ struct AppModel: ModelProtocol {
     /// This property is updated at `.start` with the corresponding value
     /// stored in `AppDefaults`.
     var sphereIdentity: String?
+    var sphereDid: Did? {
+        Did(sphereIdentity ?? "")
+    }
```

Comment:
We were doing this all over the place, may as well centralize it.

Reply:
Optional: we could consider doing this when assigning the value to the model.

Reply:
Done!